integrationtest/showcase/library/pom.xml

```diff
@@ -73,6 +77,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
             <exclusions>
+                <!-- Using a newer version of Hibernate ORM, with a different groupID -->
+                <exclusion>
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-core</artifactId>
+                </exclusion>
```

Comment:
This seems unnecessary as we've disabled this module and will only re-enable it when upgrading to Spring Boot 3, which uses Hibernate ORM 6.